packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/07-publication/01-versioning.md

```diff
@@ -1,10 +1,15 @@
 ---
-title: Versioning
+title: How to add a new version to your plugin
 ---
 
-## Versioning Your Plugin
+## How to add a new version to your plugin
 
-The Aragon OSx protocol has an on-chain versioning system built-in and distinguishes between releases and builds.
+The Aragon OSx protocol has an on-chain versioning system built-in, which distinguishes between releases and builds.
+
+- **Builds** are updates to an existing version, whereas
+- **Releases** are new version releases.
```

Comment:
I think this is a bit misleading
- Releases are major versions, which are incompatible with preexisting installations.
  - Release upgrades are not possible
  - You need to install a new, independent plugin
- Builds are minor/patch versions within a release, and they are meant for compatible upgrades only (adding a feature or fixing a bug without changing anything else)